Integrations/python/setup.py

```diff
@@ -55,6 +55,12 @@ def normalize_version(ver):
       keywords='Deephaven Development',
       packages=find_packages(exclude=['docs', 'test']),
       install_requires=['deephaven-jpy=={}'.format(__normalized_version__),
-                        'numpy', 'dill>=0.2.8', 'wrapt', 'pandas', 'numba;python_version>"3.0"',
-                        'enum34;python_version<"3.4"'],
+                        'deephaven-plugin==0.0.1.dev5',
```

Comment:
If we had deephaven-plugin-pandas external, we could specify it here instead of deephaven-plugin.

Either way, we've tended to put the hard version requirements as part of our docker/**/requirements.txt files as opposed to here in the setup.py.